packages/components/src/search/index.js

```diff
@@ -225,7 +228,10 @@ Search.propTypes = {
 	 */
 	selected: PropTypes.arrayOf(
 		PropTypes.shape( {
-			id: PropTypes.number.isRequired,
+			id: PropTypes.oneOf( [
+				PropTypes.number,
+				PropTypes.string,
+			] ).isRequired,
```

Comment:
I'm getting propType warnings for `Tag` as well as `Search`.

Reply:
Thanks - even though I had `SCRIPT_DEBUG` enabled, I wasn't getting un-minified files / seeing the warnings but I got them showing now. Fix here and in `Tag`.